### ISSUE TITLE
ERROR:root:Couldn't parse datetime string: u'2016-05-19'

### ISSUE THREAD

**Issue by elkan1788** [airbnb/superset#2768]
Hi, guys. This day I first use superset on my test environment which OS is CentOS7.2 and use python2.7.5. There all fine when I install package and run server successful. Then I very happy open browser and login. It really very nice and wonderful. But when I click slices want to explore table data it throws an exception: Couldn't parse datetime string: u'2016-05-19'. The detail error info see below:

![snapshot-2017-05-17-2.13.48.png](http://7xi700.com1.z0.glb.clouddn.com/snapshot-2017-05-17-2.13.48.png-alias)

```
2017-05-17 14:07:17,380:INFO:werkzeug:172.16.10.100 - - [17/May/2017 14:07:17] "GET /superset/welcome HTTP/1.1" 200 -
2017-05-17 14:07:18,678:INFO:werkzeug:172.16.10.100 - - [17/May/2017 14:07:18] "GET /dashboardmodelviewasync/api/read?_oc_DashboardModelViewAsync=changed_on&_od_DashboardModelViewAsync=desc HTTP/1.1" 200 -
/usr/lib/python2.7/site-packages/flask_appbuilder/forms.py:245: FlaskWTFDeprecationWarning: "flask_wtf.Form" has been renamed to "FlaskForm" and will be removed in 1.0.
  form = self(obj=obj)
2017-05-17 14:07:21,671:INFO:werkzeug:172.16.10.100 - - [17/May/2017 14:07:21] "GET /slicemodelview/list/ HTTP/1.1" 200 -
2017-05-17 14:07:24,658:INFO:werkzeug:172.16.10.100 - - [17/May/2017 14:07:24] "GET /superset/explore/table/6/?form_data=%7B%22since%22%3A%20%221%20year%20ago%22%2C%20%22slice_id%22%3A%2035%2C%20%22viz_type%22%3A%20%22cal_heatmap%22%2C%20%22row_limit%22%3A%2050000%2C%20%22metric%22%3A%20%22count%22%2C%20%22domain_granularity%22%3A%20%22month%22%2C%20%22datasource%22%3A%20%226__table%22%2C%20%22subdomain_granularity%22%3A%20%22day%22%2C%20%22granularity%22%3A%20%22day%22%2C%20%22granularity_sqla%22%3A%20%22string3%22%2C%20%22where%22%3A%20%22%22%2C%20%22until%22%3A%20%22now%22%7D HTTP/1.1" 200 -
2017-05-17 14:07:26,044:INFO:werkzeug:172.16.10.100 - - [17/May/2017 14:07:26] "GET /superset/favstar/slice/35/count/ HTTP/1.1" 200 -
2017-05-17 14:07:26,071:INFO:werkzeug:172.16.10.100 - - [17/May/2017 14:07:26] "GET /superset/datasources/ HTTP/1.1" 200 -
2017-05-17 14:07:26,171:INFO:root:SELECT ds AS __timestamp, COUNT(*) AS count 
FROM multiformat_time_series 
WHERE ds >= '2016-05-17 14:07:26.000000' AND ds <= '2017-05-17 14:07:26.000000' GROUP BY ds
 LIMIT 50000 OFFSET 0
2017-05-17 14:07:26,181:ERROR:root:Couldn't parse datetime string: u'2016-05-19'
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/superset/connectors/sqla/models.py", line 552, in query
    df = pd.read_sql_query(qry, con=engine)
  File "/usr/lib64/python2.7/site-packages/pandas/io/sql.py", line 331, in read_sql_query
    parse_dates=parse_dates, chunksize=chunksize)
  File "/usr/lib64/python2.7/site-packages/pandas/io/sql.py", line 1096, in read_query
    parse_dates=parse_dates)
  File "/usr/lib64/python2.7/site-packages/pandas/io/sql.py", line 156, in _wrap_result
    coerce_float=coerce_float)
  File "/usr/lib64/python2.7/site-packages/pandas/core/frame.py", line 1023, in from_records
    coerce_float=coerce_float)
  File "/usr/lib64/python2.7/site-packages/pandas/core/frame.py", line 5539, in _to_arrays
    data = lmap(tuple, data)
ValueError: Couldn't parse datetime string: u'2016-05-19'
2017-05-17 14:07:26,182:INFO:root:Caching for the next 86400 seconds
```

So what should I do?

> I'm a Chinese develop so is there so language setting related this exception?

**Comment by mistercrunch** [comment]
Which version are you using?

Try with `0.18.2` (latest)

`pip install superset --upgrade`

**Comment by elkan1788** [comment]
Thanks for your answer. It work fine now. But there can't understand why it not install latest package at first time use `pip install superset`, it's strange.

**Comment by mistercrunch** [comment]
Well I released `0.18.2` yesterday...
